docs/libcurl/opts/CURLOPT_PROXY_SSL_OPTIONS.3

```diff
@@ -82,6 +80,8 @@ if(curl) {
   curl_easy_cleanup(curl);
 }
 .fi
+.SH AVAILABILITY
+Added in 7.52.0
```

Comment:
One thing struck me, it would be pretty neat to have a one-line C snippet for how to test availability for all these, for easy cut-and-paste. Might get very repetitive of course, but perhaps helpful.

Reply:
I'm not following. Availability of what? The particular options? We do provide the `CURL_AT_LEAST_VERSION()` macro for easier version checks if that's what mean.